docs/source/devguides/walkthroughs/Forward-Historian-Deployment.rst

```diff
@@ -12,64 +12,52 @@ the values to the second VOLTTRON instance.
 VOLTTRON instance 1 forwards data to VOLTTRON instance 2
 --------------------------------------------------------
 
-VOLTTRON instance 1 
-~~~~~~~~~~~~~~~~~~~
-
--  Vctl shutdown –platform (if the platform is already working)
--  Volttron-cfg (this helps in configuring the volttron instance 
-   http://volttron.readthedocs.io/en/releases-4.1/core_services/control/VOLTTRON-Config.html
-
-   -  Specify the IP of the machine : tcp://130.20.*.*:22916"
-   -  Specify the port you want to use
-   -  Specify if you want to run VC(Volttron Central) here or this this instance would be controlled 
-      by a VC and the IP and port of the VC
-
-      - Then install agents like Master driver Agent with fake driver agent for the instance.
-      - Install a listener agent so see the topics that are coming from the diver agent
-      - Then run the volttron instance by : volttron –vv –l volttron.log&
-- Volttron authentication : We need to add the IP of the instance 2 in the auth.config file of the VOLTTRON agent.
-  This is done as follow :
-   -  Vctl auth-add
-   -  We specify the IP of the instance 2 and the credentials of the agent
-      (http://volttron.readthedocs.io/en/releases-4.1/devguides/walkthroughs/Agent-Authentication-Walkthrough.html?highlight=auth-add)
-   -  For specifying authentication for all the agents , we specify /.*/ for credentials as shown in 
-      http://volttron.readthedocs.io/en/releases-4.1/devguides/agent_development/index.html
-   -  This should enable authentication for all the volttron-instance based on the IP you specify here
+VOLTTRON INSTANCE 1 
+
+- ``vctl shutdown --platform`` (If VOLTTRON is already running it must be shut down before running ``volttron-cfg``).
+- ``volttron-cfg`` - this helps in configuring the VOLTTRON instance(http://volttron.readthedocs.io/en/releases-4.1/core_services/control/VOLTTRON-Config.html).
+
+  - Specify the IP of the machine : ``tcp://130.20.*.*:22916``.
+  - Specify the port you want to use.
+  - Specify if you want to run VC ( VOLTTRON Central) here or this this instance would be controlled by a VC and the IP and port of the VC.
+- Then start the VOLTTRON instance by : ``volttron –vv –l volttron.log&``.
+- Then install agents like Master driver Agent with fake driver agent for the instance.
```

Comment:
change the start to volttron -vv &> volttron.log &
This will redirect both standard out and standard error to the file so we don't have output from stderr being printed on the console.

Reply:
Changed to  volttron -vv & > volttron.log&